packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -112,7 +111,6 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		useBlockRefProvider( clientId ),
 		useFocusHandler( clientId ),
 		useEventHandlers( { clientId, isSelected } ),
-		useZoomOutModeExit(),
```

Comment:
Good question, why was it added here? It's good to avoid adding hooks to every block if we can capture the event higher up.

Reply:
And the other one in the `editor` package was added here: https://github.com/WordPress/gutenberg/pull/65963

Reply:
@andrewserong Any reason you left the other one in place?

Reply:
Thanks for the ping! Nope, I left it there by accident as there were a couple of iterations on the approach. Thanks for following up!